python/services/evaluationservice/dmod/evaluationservice/kill_runner.py

```diff
@@ -24,10 +26,18 @@ def host(self) -> typing.Optional[str]:
     def port(self) -> typing.Optional[int]:
         return self.__port
 
+    @property
+    def username(self) -> typing.Optional[str]:
+        return self.__username
+
     @property
     def password(self) -> typing.Optional[str]:
         return self.__password
 
+    @property
+    def db(self) -> int:
+        return self.__db
```

Comment:
A `db` value is a redis construct, almost like a RDMS schema. Redis comes with around 16 different `db`s available. If you state that you want to connect to `localhost:8675` and `db` 3 and _**I**_ want to connect to `localhost:8675` and `db` 9, our data will not come into conflict. The default is `0` and it is _usually_ recommended to use that and use syntactic structures to keep applications apart (`PRODUCT_ONE::KEY` and `PRODUCT_TWO::KEY` instead of just `KEY`), but using a different `db` value comes in handy from time to time, such as when different versions of the same applications are being tested with the same redis server.

Would it help to have that as a docstring?

Reply:
Thanks for the explantation! Learn something new everyday! Nope! Sounds like something that is pretty commonly used or at least is common knowledge if you are more familiar with redis.